features/FxA-45-device-registration-api/README.md

```diff
@@ -81,11 +71,6 @@ in the future.
   and be signed out of all devices,
   including the current device.
 
-* As a signed-in user,
-  I want to update my device details
-  and see the newly-changed details
-  automatically reflected in all open instances of the devices view.
-
```

Comment:
Mostly the second one I think, but also a bit of the first because we haven't fully settled on the approach for sending unsolicited web channel messages from the browser to the content server (I'm hoping to chat to people about this next week a bit).